exercises/I-floats/exercise.js

```diff
@@ -1,2 +1,9 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+let totalClass = numberOfMentors + numberOfStudents;
+let percentStudents = numberOfStudents / totalClass;
+let percentMentors = numberOfMentors / totalClass;
```

Comment:
You may want to add the *100 part to the percent variables rather than line 7 and 9 as it currently isn't a percentage.